csharp/HOLMS.Platform/HOLMS.Platform/Types/Extensions/IAM/Department.cs

```diff
@@ -1,45 +1,29 @@
 ﻿namespace HOLMS.Types.IAM {
     public partial class Department {
-        public Department(DepartmentIndicator id, string name, bool isHousekeeper, bool capStaffMgmt, bool capGroupMgmt,
-            bool capHousekeepingMgmt, bool capAccountingReports, bool capManagementReports,
-            bool capWorkstationParameters, bool capSaveDisplaySettings, bool capLodgeicReset,
-            bool capEventsWaitlistSetup, bool capSystemSettings, bool capYieldManagement,
-            bool capHoldMgmt, bool capGuestBlacklistAddRemove, bool capOverbookRooms,
-            bool capOverrideRates, bool capCallAccountingAdvancedSetup, bool capGiftShopsalesSetup,
-            bool capCreditCardSetup, bool capCreditCardWorksheet, bool capCreditCardCaptureSettle,
-            bool capCreditCardReporting, bool capExpediaQCMgmt, bool capWebSetup) {
+        public Department(DepartmentIndicator id, string name, bool isHousekeeper, bool capGetSupplySnapshot, bool capGetRoomTypes,
```

Comment:
I think we're talking past each other.

When I said "this sucks", I was referring very narrowly to the problem of passing a bunch of positional params. I wasn't commenting on the way it's stored, passed over the wire (as a DTO), serialized, etc., though I'm sure those could all use work. Let's address one problem at a time.

The immediate concern is the need to pass in over a dozen bools, in the right order, in order for this constructor to function. This is almost guaranteed to introduce bugs later, so I was proposing rolling these parameters into an object, e.g. a `AuthorizationSet` or `CapabilitySet`, that gets passed into the department object, in lieu of all these positional parameters.

Let's make that fix first, then we can address the other stuff in subsequent work.